Testscripts/Linux/validate-da.sh

```diff
@@ -0,0 +1,251 @@
+#!/usr/bin/env bash
+#######################################################################
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# validate-da.sh
+# Description:
+#    Validate the installation and uninstallation of the Dependency Agent.
+#    Validate the enable and disable of the Dependency Agent.
+#######################################################################
+
+set -e
+set -x
+
+# Source utils.sh
+. utils.sh || {
+    echo "ERROR: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+
+# Get distro information
+GetDistro
+
+readonly dir_list=(/opt/microsoft/dependency-agent /var/opt/microsoft/dependency-agent /etc/opt/microsoft/dependency-agent /lib/microsoft-dependency-agent /etc/init.d/microsoft-dependency-agent)
+
+readonly da_pid_file="/etc/opt/microsoft/dependency-agent/config/DA_PID"
+readonly uninstaller="/opt/microsoft/dependency-agent/uninstall"
+
+function test_case_cleanup(){
+    [ -f "$uninstaller" ] && eval $uninstaller
```

Comment:
I don't understand why the `eval` is needed, nor why double quotes are needed for one use of `$uninstaller`, but not the other.

Reply:
Removed the `eval` and the quotes. I initially used `eval` because I was testing in my local linux machine on how run a command stored in variable. 

I was testing the following command: 
test="cd Desktop/lisa"

For this case, `$test` wasn't working so I found out that I had to use `eval $test` to run the command. 

It was small confusion :)